src/goGenerateTests.ts

```diff
@@ -101,7 +101,14 @@ export async function generateTestCurrentFunction(): Promise<boolean> {
 	if (funcName.includes('.')) {
 		funcName = funcName.split('.')[1];
 	}
-	return generateTests({ dir: editor.document.uri.fsPath, func: funcName }, getGoConfig(editor.document.uri));
+
+	const currentFilePath = editor.document.fileName;
+	if (!currentFilePath.endsWith('.go')) {
+		vscode.window.showInformationMessage('Cannot toggle test file. File in the editor is not a Go file.');
+		return;
+	}
+
+	return generateTests({ dir: editor.document.uri.fsPath, func: funcName, testFile: currentFilePath.endsWith('_test.go') }, getGoConfig(editor.document.uri));
```

Comment:
shouldn't we set this boolean for the other 2 cases where tests are generated for a file/package? i.e in `generateTestCurrentFile` and `generateTestCurrentPackage`?